app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java

```diff
@@ -24,6 +26,20 @@
 import android.widget.Toast;
 
 import com.afollestad.materialdialogs.MaterialDialog;
+import com.google.android.gms.auth.api.credentials.Credential;
+import com.google.android.gms.auth.api.credentials.CredentialRequest;
+import com.google.android.gms.auth.api.credentials.CredentialRequestResponse;
+import com.google.android.gms.auth.api.credentials.Credentials;
+import com.google.android.gms.auth.api.credentials.CredentialsClient;
+import com.google.android.gms.auth.api.credentials.CredentialsOptions;
+import com.google.android.gms.common.ConnectionResult;
+import com.google.android.gms.common.GoogleApiAvailability;
+import com.google.android.gms.common.api.ApiException;
+import com.google.android.gms.common.api.CommonStatusCodes;
+import com.google.android.gms.common.api.GoogleApiClient;
+import com.google.android.gms.common.api.ResolvableApiException;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
```

Comment:
@jaindiv26 I'm concerned about all those imports. Can't they be made conditional @Karljoones @ross-holloway94 ?

Reply:
@teolemon Java does not support conditional imports, so that would not be possible.

Reply:
so not possible to bind them to a non free flavour ?

Reply:
@teolemon Yes, it will not be possible.